docs/components/SidebarSectionLink.js

```diff
@@ -10,7 +10,7 @@ type Props = {|
 export default function SideBarSectionLink({ componentName }: Props): Node {
   return (
     <Box role="listitem">
-      <NavLink to={`/${componentName}`}>
+      <NavLink href={`/${componentName.replace(/ /g, '_').replace(/'/g, '')}`}>
```

Comment:
nit: this could use a helper function with an explanatory name (and comment), something like
```js
// Next.js uses underscores for spaces in URLs
const spacesToUnderscores = (str) => str.replace(/ /g, '_').replace(/'/g, '');

return (
    <Box role="listitem">
      <NavLink href={spacesToUnderscores(componentName)}>
```

Reply:
@dangerismycat added the comment